src/cgame/cg_players.cpp

```diff
@@ -3488,10 +3488,14 @@ void CG_PlayerDisconnect( vec3_t org )
 
 centity_t *CG_GetLocation( vec3_t origin )
 {
-	float     bestlen, len;
+	static std::vector<int> validEntities;
+	validEntities.reserve( MAX_LOCATIONS );
+	validEntities.resize( 0 );
 
-	centity_t *best = nullptr;
-	bestlen = 3.0f * 8192.0f * 8192.0f;
+	// Shared with the engine.
```

Comment:
I find this comment confusing. It sounds like it means it is in shared memory or something

BTW why did you make the vectors `static`?

Reply:
Cleaned-up.